addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -178,7 +178,7 @@ PortalChatter.include({
         return {
             mes_index: messageIndex,
             publisher_id: this.options.partner_id,
-            publisher_avatar: _.str.sprintf('/web/image/%s/%s/image_128/50x50', 'res.partner', this.options.partner_id),
+            publisher_avatar: _.str.sprintf('/web/image/%s/%s/avatar_128/50x50', 'res.partner', this.options.partner_id),
```

Comment:
I know this is currently minimal diff but I would change this a bit, it's confusing to have `res.partner` has a `%s` string when it's a static value.